### ISSUE TITLE
Training code released?

### ISSUE THREAD

**Issue by tungyen** [Sojwal5/Recipe_Generation_From_Food_Image#1]
Thank you for this nice work. I wonder if the code for training the model could be released? Thank you very much!

**Comment by Sojwal5** [comment]
Thank you for your kind words! I'm glad you found the work helpful.
Unfortunately, at this time, I am unable to release the code for training
the model. However, I'd be happy to discuss the methods or provide further
insights if you're interested. Please feel free to ask!

**Comment by tungyen** [comment]
Yes, I am interested what kinds of datasets you use for training. And could you mention more details when training? Like the optimizer, loss function, and something else that you don't mind sharing. Thank you very much!

**Comment by tungyen** [comment]
And I also want to ask your insight about how to predict calories based on this model as extension? Now your work can predict ingredient, but I also interested in predicting calories as well!
